homeassistant/components/zha/__init__.py

```diff
@@ -333,6 +356,13 @@ def device_removed(self, device):
                     'in_clusters',
                     join,
                 )
+                if cluster.cluster_id in EVENTABLE_CLUSTERS:
```

Comment:
The two loops that we're modifying here and below look very similar. I suggest we break those out into a small helper method that takes parameters, `endpoint`, `profile_clusters`, `device_key` `cluster_in_out`, `join`, for the variables.

Reply:
will do